### PR TITLE
added copy-button

### LINKED ISSUE #337: Copy feature Enhancement
**Current Behavior**

If we want to copy the command for any OS Distribution, we have to select the command and then copy. Which might some time-consuming. 

**Desired Behavior**

We can change this by creating a copy button on the top-right corner so that if we want to copy the command, the only thing we do is click on the copy button. It will be very easy for developers.
---
image

**Contributor Resources**

- See the [Contributing to getnighthawk.dev Website](https://github.com/layer5io/getnighthawk/blob/master/CONTRIBUTING.md)
- See the Project Site section of [GetNighthawk project doc](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w)
- 🎨 Wireframes and designs for Nighthawk Site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?type=design&node-id=5046-0&mode=design&t=UYVaGhic2LY8EXee-0)
- Meshery UI Nighthawk (Performance) in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?type=design&node-id=2-5814&mode=design&t=Gt14dmRRlpL09apZ-0)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by omesh-omg**
**Description**
added copy button
This PR fixes #337 

https://github.com/layer5io/getnighthawk/assets/64972958/0fe4983b-0d01-4cac-8b1c-af6109140fdb

I have added the button using javascript dom manipulation by searching all the code blocks and 
adding the button to them in the code and than there is a CSS file for button, and also imported clipboard.min.js for copy function.  

**[Signed commits](https://github.com/layer5io/getnighthawk/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Savio629** [comment]
@omesh-omg 
You need to change the 'Copy' text to Copy icon
Also remove the vertical scrollbar arised from your code
image

**Review by Savio629 — CHANGES_REQUESTED** [review]
Please revert back with unnecessary changes.

**Comment by omesh-omg** [comment]
> Please revert back with unnecessary changes.

@Savio629 okay

File: docs/assets/css/copy-button.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    top: 0;
+    right: 0;
+    padding: 5px 10px;
+    font-size: 14px;
+    cursor: pointer;
+    border: 1px solid #009E7F;
+    border-radius: 3px;
+    background-color: #009E7F;
+    color: #fff;
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```
Comment (Savio629) [inline docs/assets/css/copy-button.css:12]: Replace the font-family with the one that is already in use.

Comment (omesh-omg) [reply docs/assets/css/copy-button.css:12]: okay see for all that you can suggest,
 anything you else you find  that needs attention.

**Comment by omesh-omg** [comment]
> @omesh-omg You need to change the 'Copy' text to Copy icon Also remove the vertical scrollbar arised from your code image

is there any specific icon for the copy button ? it can't see that in the Figma file of nighthawk, 
In meshery website there is a copy icon but it is in different theme , so we can't use it here

**Pull request closed without merging**